cluster-provision/manifests/linux-bridge-config-nftables.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
```

Comment:
No need for this file we can do the same as openshift-sdn use chroot and mount the host into our utility pod.

please take a look on this https://bugzilla.redhat.com/show_bug.cgi?id=1708500

Reply:
Neat, thanks!